coremltools/models/model.py

```diff
@@ -16,6 +16,8 @@
 from .utils import macos_version as _macos_version
 from .utils import save_spec as _save_spec
 from ..proto import Model_pb2 as _Model_pb2
+from coremltools.models.deprecation import deprecated
+import warnings
```

Comment:
You probably want to hide this import. Since it's only used in one function, I would probably just import it at the top of that function, in which case you don't need to hide it.

Reply:
moved it down below